tests/lightning_tensor/test_gates_and_expval.py

```diff
@@ -73,7 +73,7 @@ def circuit_ansatz(params, wires):
         qml.matrix(qml.PauliX([wires[1]])), control_wires=[wires[0]], wires=wires[1]
     )
     qml.DiagonalQubitUnitary(np.array([1, 1]), wires=wires[2])
-    qml.MultiControlledX(wires=[wires[0], wires[1], wires[3]], control_values=[wires[0], wires[1]])
+    qml.MultiControlledX(wires=[wires[0], wires[1], wires[3]])
```

Comment:
```suggestion
    qml.MultiControlledX(wires=[wires[0], wires[1], wires[3]], control_values=[0, 1])
```
By checking the `test_gates_and_expval.py` it seems the mod here should like this